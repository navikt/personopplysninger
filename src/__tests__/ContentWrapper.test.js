import * as React from 'react';
import ContentWrapper from 'js/ContentWrapper';
import { IntlProvider } from 'react-intl';
import wrapIntl from 'js/IntlTestHelper';
import initialState from '../js/initialState';
const ReactTestRenderer = require('react-test-renderer');

test('render ContentWrapper with content', () => {
  const component = ReactTestRenderer.create(wrapIntl(<ContentWrapper userInfo={initialState} />));
  expect(component.toJSON()).toMatchSnapshot();
});
