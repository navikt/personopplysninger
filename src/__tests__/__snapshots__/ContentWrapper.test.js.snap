// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ContentWrapper with content 1`] = `
<div>
  <div
    className="BoxContainer"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Personalia
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <img
          alt=""
          className="box-icon"
          src="woman.png"
        />
        <div
          className="Box"
        >
          <ul
            className="personalia-list"
          >
            <li>
              <span
                className="title"
              >
                <span>
                  Fornavn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Etternavn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Fødselsnummer
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Utenlandsk ID
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Telefon
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Kontonummer
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  e-post
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Språk
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Statsborgerskap
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Personstatus
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Fødested
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Spesialregistrering
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Sivilstand
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Sikkerhetstiltak
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Kjønn
                </span>
              </span>
              <span
                className="content"
              >
                
              </span>
            </li>
            <li>
              <span
                className="title"
              >
                <span>
                  Tilpasset kommunikasjon
                </span>
              </span>
              <span
                className="content"
              >
                x
              </span>
            </li>
          </ul>
          <div
            className="box-footer"
          >
            <span>
              Kilde: NAV, Skatteetaten, Folkeregisteret, Kontakt- og reservasjonsregisteret
            </span>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Adresse
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="Box"
        >
          <h3>
            Adresse
          </h3>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Familierelasjoner
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="Box"
        >
          <h3>
            Familierelasjoner
          </h3>
        </div>
      </div>
    </div>
  </div>
  <div
    className="BoxContainer"
  >
    <div
      className="BoxWithHeader"
    >
      <h1>
        Arbeidsforhold
      </h1>
      <div
        className="icon-box-wrapper"
      >
        <div
          className="Box"
        >
          <h3>
            Arbeidsforhold
          </h3>
        </div>
      </div>
    </div>
  </div>
</div>
`;
